src/libs/ReportUtils.ts

```diff
@@ -4936,7 +5002,9 @@ function buildOptimisticModifiedExpenseReportAction(
  * @param transactionThreadID - The reportID of the transaction thread
  * @param movedToReportID - The reportID of the report the transaction is moved to
  */
-function buildOptimisticMovedTrackedExpenseModifiedReportAction(transactionThreadID: string, movedToReportID: string): OptimisticModifiedExpenseReportAction {
+function buildOptimisticMovedTrackedExpenseModifiedReportAction(delegate: string, transactionThreadID: string, movedToReportID: string): OptimisticModifiedExpenseReportAction {
```

Comment:
I don't see Track Expense actions are required for this feature. IMO, we should add this. But need confirmation to sure